rele/client.py

```diff
@@ -49,6 +50,22 @@ def __init__(self):
                 credentials=settings.RELE_GC_CREDENTIALS)
 
     def publish(self, topic, data, blocking=False, **attrs):
+        """Publish data to Pub/Sub.
+
```

Comment:
I would suggest to add a sentence about the `published_at` time stamp attribute that we add.

Reply:
@andrewgy8 I've changed the doc with your suggestion. Let me know if it's ok.